src/mixxx.cpp

```diff
@@ -1510,6 +1515,27 @@ void MixxxMainWindow::initActions()
     }
 }
 
+void MixxxMainWindow::slotUpdateWindowTitle(TrackPointer pTrack)
+{
+    QString title;
+    if (pTrack) {
+        QString trackTitle = pTrack->getTitle();
+        QString trackArtist = pTrack->getArtist();
+
+        //Displaying title as "Track Title" by "Artist" - "Current Version"
+        //Or "Track Title - "Current Version" in case the artist field is empty
+        if (trackArtist.size() != 0) {
+            title = trackArtist + " - " + trackTitle;
+        } else {
+            title = trackTitle;
+        }
+        title = title + " - " + m_MixxxVersion;
```

Comment:
I think you should use another sign then `-` to seperate the mixxx version. It is a little bit confusing to see the minus twice. Maybe write the string like this `artist: song  -  Mixxx`. (note the two spaces around -).